src/module-4/filters-list/index.js

```diff
@@ -1,11 +1,68 @@
 export default class FiltersList {
-  constructor ({
-     title = '',
-     list = []
+  constructor({
+    title = '',
+    list = []
   } = {}) {
     this.title = title;
     this.list = list;
 
-    // ... your logic
+    this.addFilter();
+  }
+
+  addFilter() {
+    this.render();
+  }
+
+  getTemplate() {
+    let filterName =
+      `<div class="filter__name">
+      <div class="filter__name-inner">
+        ${this.title}
+      </div>
+    </div>\n`;
+
+    let filters = '';
+    let check = "";
+
+    for (let i = 0; i < this.list.length; i++) {
+
+      if (this.list[i].checked) {
+        check = "checked";
+        console.log("check " + i);
+      }
+
+      filters +=
+        `<div class="filter__input-outer">
+        <input type="checkbox" id="${this.list[i].value}" name="${this.title}" value='${this.list[i].value}' ${check}><label
```

Comment:
```suggestion
        <input type='checkbox' id='${this.list[i].value}' name='${this.title}' value='${this.list[i].value}' checked='${this.list[i].checked}'><label
```  
теперь можно не использовать переменную check. Также можно создать переменную для `this.list[i]`. Или вместо цикла for испoльзовать map по принципу `const filters = this.list.map(item => return template).join('')